### PR TITLE
javascript3-week2/srushti

### PR DESCRIPTION
**Pull request by srushtisalke**
Completed Javascript week 2 homework.
Need a review.

### CODE REVIEW

**Comment by srushtisalke** [comment]
Hi @shpomp,
I am missing PR on this submission as well. Can you ask someone to review it? 

BR Srushti

File: 3_javascript/javascript3/week2/script.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+let exchangeRates;
+fetch("https://open.er-api.com/v6/latest/USD")
+  .then((response) => response.json())
+  .then((data) => {
+    exchangeRates = data.rates;
+    dropdownsFunction();
+  });
+
+function dropdownsFunction() {
+  const currencies = Object.keys(exchangeRates);
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/script.js:15]: Well done getting the currencies using ```Object.keys()```

File: 3_javascript/javascript3/week2/script.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  fromCurrencySelect.innerHTML = createOptions(currencies, "EUR");
+  toCurrencySelect.innerHTML = createOptions(currencies, "DKK");
+
+  fromCurrencySelect.value = "EUR";
+  toCurrencySelect.value = "DKK";
+
+  amountInput.addEventListener("input", updateResult);
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/script.js:22]: Good idea to add event listeners.

File: 3_javascript/javascript3/week2/script.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+fetch("https://open.er-api.com/v6/latest/USD")
+  .then((response) => response.json())
+  .then((data) => {
+    exchangeRates = data.rates;
+    dropdownsFunction();
+  });
+
+function dropdownsFunction() {
+  const currencies = Object.keys(exchangeRates);
+  fromCurrencySelect.innerHTML = createOptions(currencies, "EUR");
```
Comment (balance3840) [inline 3_javascript/javascript3/week2/script.js:16]: Good idea to create the function ```createOptions``` to avoid repetitive code.

**Review by balance3840 — APPROVED** [review]

**Comment by srushtisalke** [comment]
As no changes required from PR.

**Pull request closed without merging**